src/diff/index.js

```diff
@@ -298,6 +298,7 @@ function diffElementNodes(
 	}
 
 	if (dom == null) {
+		isHydrating = false;
```

Comment:
Might be cheaper to reuse the `null` allocation of line 313 and set `isHydrating` to null instead WDYT?

Reply:
I think we could just move this to 212 without changing its type to `null`. Terser will be able to merge the statements better there.